libnetwork/networkdb/networkdb.go

```diff
@@ -422,8 +422,11 @@ type TableElem struct {
 // GetTableByNetwork walks the networkdb by the give table and network id and
 // returns a map of keys and values
 func (nDB *NetworkDB) GetTableByNetwork(tname, nid string) map[string]*TableElem {
+	nDB.RLock()
+	root := nDB.indexes[byTable].Root()
```

Comment:
You intentionally combined the fix-up from d5c370dee61e87da0a6b317722cbc8c0de8664dc in this commit?

Reply:
Yes.